beacon-chain/blockchain/forkchoice/process_block.go

```diff
@@ -157,6 +164,25 @@ func (s *Store) verifyBlkFinalizedSlot(b *ethpb.BeaconBlock) error {
 	return nil
 }
 
+// saveNewValidator saves newly added validator index from state to db.
+func (s *Store) saveNewValidator(ctx context.Context, preStateValidatorCount int, postState *pb.BeaconState) error {
+	postStateValidatorCount := len(postState.Validators)
+	if preStateValidatorCount != postStateValidatorCount {
+		for i := preStateValidatorCount; i < postStateValidatorCount; i++ {
+			pubKey := postState.Validators[i].PublicKey
+			if err := s.db.SaveValidatorIndex(ctx, bytesutil.ToBytes48(pubKey), uint64(i)); err != nil {
+				return errors.Wrapf(err, "could not save activated validator: %d", i)
+			}
+			log.WithFields(logrus.Fields{
```

Comment:
I'm leaning towards `Info` here. Not only it helps a lot with debugging, this is quite an important log, it would be nice to make it "seen"